### PR TITLE
Code inspection and cleanup

### PR DESCRIPTION
**Pull request by piraka9011**

### CODE REVIEW

File: sysroot_creator/scripts/create_cc_sysroot.py [hunk 1]
```diff
@@ -298,7 +298,17 @@ def setup_sysroot_environment(
 
 def create_arg_parser():
     """Parse command line arguments."""
-    parser = argparse.ArgumentParser()
+    example_text = '''Example usage:
+    
+    python create_cc_sysroot.py --arch armhf --os debian
```
Comment (prajakta-gokhale) [inline sysroot_creator/scripts/create_cc_sysroot.py]: s/`python`/`python3`/

File: README.md [hunk 2]
```diff
@@ -44,23 +45,24 @@ python3 -m pip install --user -U  colcon-common-extensions rosdep vcstool
...
-# Create a directory to store qemu assets
-mkdir -p sysroot/qemu-user-static
-cp /usr/bin/qemu-* sysroot/qemu-user-static 
+cd /path/to/sysroot
+# Create a directory to store ROS/qemu assets
+mkdir qemu-user-static ros2_ws
+cp /usr/bin/qemu-* qemu-user-static 
 # Copy ROS Sources
-cp -r ~/ros2_ws/src sysroot
+cp -r ~/ros2_ws/src ros2_ws
```
Comment (prajakta-gokhale) [inline README.md:53]: The dockerfile currently expects to find ros2 sources in [<sysroot_path>/src/](https://github.com/aws-ros-dev/cross_compile/blob/cc-sysroot-creator/sysroot_creator/scripts/sysroot/Dockerfile_workspace#L16). Let's keep it consistent with that?

File: tests/test_sysroot_creator.py [hunk 3]
```diff
@@ -20,17 +20,16 @@
 from glob import glob
 from pathlib import Path
 from types import SimpleNamespace
-from typing import Dict
```
Comment (prajakta-gokhale) [inline tests/test_sysroot_creator.py:23]: Good catch, thank you!

**Review by prajakta-gokhale — APPROVED** [review]
Looks good to me. Please address the documentation comments and merge.

**Review by thomas-moulard — APPROVED** [review]

**Pull request merged**